build-support/bin/readme_api.py

```diff
@@ -96,9 +98,11 @@ def field_value_from_api_response(cls, field_name: str, field_type: type[F], val
             return super().field_value_from_api_response(field_name, field_type, val)
 
     def __str__(self):
-        return (
-            f"DocRef(id={self.id}, slug={self.slug}, title={self.title}, "
-            f"order={self.order}, hidden={self.hidden})"
+        return softwrap(
+            f"""
+            DocRef(id={self.id}, slug={self.slug}, title={self.title},
+            order={self.order}, hidden={self.hidden})
+            """
         )
```

Comment:
To be honest, I am not sure this is a win. More lines + import. We seem unlikely to add onto this code also